src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/HerculesPickaxe.java

```diff
@@ -50,13 +50,13 @@ public HerculesPickaxe(ItemGroup itemGroup, SlimefunItemStack item, RecipeType r
                 switch (mat) {
                     case IRON_ORE:
                         drops.add(new CustomItemStack(SlimefunItems.IRON_DUST, 2));
-                        break;
+                        return;
                     case GOLD_ORE:
                         drops.add(new CustomItemStack(SlimefunItems.GOLD_DUST, 2));
-                        break;
+                        return;
                     default:
                         for (ItemStack drop : e.getBlock().getDrops(tool)) {
```

Comment:
I think this loop is useless. You are adding to `drops` item which can drop by default without adding it.

Reply:
> I think this `for` loop is useless. You are adding to `drops` items which can drop by default without adding them to list.

Tested it ingame, drops are cancelled somewhere, if the default fallback is removed and a diff ore is mined which has no dust associated with them it will not drop anything using the hercules pickaxe. So its better to have the for loop to do its default drop

Reply:
It should drop vanilla items by default if  `drops` list is empty
https://github.com/Slimefun/Slimefun4/blob/1773ae0a4a33b28c56ee031f24e7297145db7e65/src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BlockListener.java#L169-L176

Reply:
I Might as well test it out without the default loop ingame, Didn't touched the code since it was there before this pr